src/Application/Actions/CreatePerson.php

```diff
@@ -108,6 +110,45 @@ protected function action(): Response
 
         $person = $this->personRepository->persist($person);
 
+        // After persisting the Person, send them a registration success email
+        try {
+            $this->httpClient = new Client([
+                'timeout' => $this->settings->get('apiClient')['global']['timeout'],
+            ]);
+
+            $requestBody = $person->toMailerPayload();
+
+            $registrationEmailResponse = $this->httpClient->post(
+                $this->settings->get('apiClient')['mailer']['baseUri'] . '/v1/send',
+                [
+                    'json' => $requestBody,
+                    'headers' => [
+                        'x-send-verify-hash' => $this->hash(json_encode($requestBody)),
+                    ],
+                ]
+            );
+        }
+
+        catch (RequestException $ex) {
```

Comment:
Imports are missing from this and the 2 thrown exceptions, meaning none of this can work. This should all be test covered so we know that it does work.